rfcs/_templates/rfc-template-0000.md

```diff
@@ -0,0 +1,56 @@
+**Feature Name** A desciptive name of the proposal
+
+**Start Date** The date the proposal is submitted
+
+**Issue** Link to the Bugzilla or Github issue
+
+# Summary
```

Comment:
I think we should be allowed to remove some sections entirely if they don't apply, this would make the process feel less "heavy" and less "red tape-y". Of course for big RFCs we will fill out everything, but for smaller things we might remove a couple of sections.

How does everyone feel about this?

Reply:
Yeah, that makes sense. However I think we need to make some sections mandatory - a `Summary` and either an `Explanation` or `Technical Specification` must be there in order to give enough information to start a discussion.

Reply:
Yeah, that makes sense. However I think we need to make some sections mandatory - a `Summary` and either an `Explanation` or `Technical Specification` must be there in order to give enough information to start a discussion.